client/components/AggregatedReviews.jsx

```diff
@@ -1,9 +1,75 @@
 import React from 'react';
-import style from './styles/AggregatedStyle.css';
+import $ from 'jquery';
+import './styles/AggregatedStyle.css';
+import emptyStar from './styles/icons/star-clear.svg';
+import halfStar from './styles/icons/star-half.svg';
+import fullStar from './styles/icons/star.svg';
 
 const AggregatedReviews = (props) => {
+  const renderStars = function (rating, outerDiv) {
```

Comment:
So this is a pretty clever approach to this problem, but I'm wondering if there's a cleaner way to solve it that doesn't require manipulating the DOM directly:

-What if you pushed your image tags as jsx to an array that you returned from this function? I think you may be able to achieve the same goal that way without having to worry about conditional rendering